File: Data Structures & Algorithms/special-array-i/submission-0.cs
public class Solution {
    public bool IsArraySpecial(int[] nums) {
        for (var i=0; i < nums.Length - 1;i++)
        {
            var num1 = nums[i];
            var num2 = nums[i+1];

            if ((num1 + num2) % 2 == 0)
                return false;
        }
        return true;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
    
    #nx + (0 + 1 + 2 + (b-1)) = sum
    #nx + b(b-1)/2 = 
    #hash = 2, 20, 4, 10, 3,4,5
    #
        nset = set(nums)
        longest = 0

        for n in nums:
            if (n -1) not in nset:
                length = 0
                while (n + length) in nset:
                    length += 1
                longest = max(length, longest)

        return longest        
